components/feature/accounts-push/src/main/java/mozilla/components/feature/accounts/push/FxaPushSupportFeature.kt

```diff
@@ -67,6 +65,9 @@ class FxaPushSupportFeature(
      * A unique scope for the FxA push subscription that is generated once and stored in SharedPreferences.
      *
      * This scope is randomly generated and unique to the account on that particular device.
+     * (why this uuid? Note it is *not* reset on logout!)
```

Comment:
These comments are why I left this PR as a draft - please let me know what it should say instead :)